test-basic/ssl-min-allow-tls-test.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2025 MarkLogic Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+let testconfig = require('../etc/test-config.js');
+let should = require('should');
+let marklogic = require('../');
+const { exec } = require('child_process');
+let db = marklogic.createDatabaseClient(testconfig.restWriterConnection);
```

Comment:
We need two clients here. One with TLSv1.3 enabled (use this in the existing tests), and one with TLSv1.2 enabled.

Then there should be another test with an attempt to use the TLSv1.2 client with a DB set to minimum TLSv1.3, and expect an exception.